career/general_personality.md

```diff
@@ -0,0 +1,93 @@
+
+Bootcamp graduates are self-motivated, embrace challenge and are intelligent problem solvers with a broad set of skills and experience. They have the self-confidence to make an informed choice about pivoting into a new profession and planning for the long term. They are also tough - tough enough to make it through a bootcamp, an endeavor that not only requires them to learn a completely new skill-set but quite often reassesses their whole approach towards how they function as professionals.
+
+Being able to do all that takes a lot of courage, determination, motivation and .., yes, passion.
+
+### Describe a time when you were under extreme pressure and your application wasn’t working. What did you do?
+
+Grace under fire
+
+It may not seem like it to the outsider, but programming can be an extremely stressful profession. When deadlines are tight and nothing seems to be working, it's easy to freeze up. We argue that the ideal candidate should be able to handle even the most stressful situations calmly and, most importantly, be able to continue working. The challenge is to be able to showcase situations like that from your previous career or the camp itself and tell a compelling story about how you dealt with overcoming the obstacles.
+
+### Explain how your favorite application works in a way a layman would understand.
+
+People skills
+ Programmers generally aren’t expected to have heavy-duty customer service talents. It seems to be understood at the outset that they’ll mostly be sitting in front of a computer all day. Yet programmers are regularly expected to interact with managers, co-workers, and clients, so the ability to work well with others is a must. This is especially true if your programmers are occasionally asked to participate in client meetings and explain how a system works.
+
+### Tell us about a time when you saved time by automating a process.
+
+Laziness (?!?)
+
+Larry Wall, the original author of, The Perl Programming Languages, describes three good qualities in a good programmer: laziness, impatience and hubris. Laziness may sound like a bad trait for any employee to have, but IT managers have said that if you want to find the best way to do something, ask a lazy person. Chances are, that person will have found the quickest, most efficient way to do it. A company’s programming staff can often find a way to automate processes, saving time and money.
```

Comment:
`Larry Wall, the original author of, The Perl Programming Languages, describes three good qualities` 
`Larry Wall, the original author of The Perl Programming Languages, describes three good qualities`